Add Asset Store configuration docs (#3935)
kyma-project/kyma: docs/asset-store/docs/05-04-asset-upload-service.md

---
title: Asset Store Upload Service sub-chart
type: Configuration
---

To configure the Asset Store Upload Service sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.

## Configurable parameters

This table lists the configurable parameters, their descriptions, and default values:

| Parameter | Description | Default value |
|-----------|-------------|---------------|
| **replicaCount** | Defines the number of replicas of the service. | `1` |

> | **virtualservice.enabled** |  Enables to use an external service. | `false` |

Reviewer: please specify that once you enable virtualservice it is available under https://{VIRTUALSERVICE_NAME}.{DOMAIN}/minio/, for example https://asset-upload-service.kyma.local/

Author: Changed to:
If you enable the **virtualservice**, it is available at `https://{VIRTUALSERVICE_NAME}.{DOMAIN}/`, for example `https://asset-upload-service.kyma.local/`.

---
title: Asset Store Upload Service sub-chart
type: Configuration
---

To configure the Asset Store Upload Service sub-chart, override the default values of its `values.yaml` file. This document describes parameters that you can configure.

>**TIP:** To learn more about how to use overrides in Kyma, see the following documents: 
>* [Helm overrides for Kyma installation](/root/kyma/#configuration-helm-overrides-for-kyma-installation)
>* [Sub-charts overrides](/root/kyma/#configuration-helm-overrides-for-kyma-installation-sub-chart-overrides)

## Configurable parameters

This table lists the configurable parameters, their descriptions, and default values:

| Parameter | Description | Default value |
|-----------|-------------|---------------|

> | **replicaCount** | Defines the number of replicas of the service. | `1` |

Reviewer 2: or service replicas - as you prefer

> | **virtualservice.enabled** |  Enables to use an external service. If you enable the **virtualservice**, it is available at `https://{VIRTUALSERVICE_NAME}.{DOMAIN}/`, for example `https://asset-upload-service.kyma.local/`. | `false` |

Reviewer 2: ```suggestion
| **virtualservice.enabled** |  Enables the use of an external service. If you activate **virtualservice**, it is available at `https://{VIRTUALSERVICE_NAME}.{DOMAIN}/`, for example at`https://asset-upload-service.kyma.local/`. | `false` |
```
